examples/make_bg_cube_models_true-reco_test.py

```diff
@@ -0,0 +1,217 @@
+"""
+Script to produce true and reco background cube models.
+
+The script uses the same model (except for absolute normalization) to
+produce a true cube bg model and a reco cube bg model. The models can
+be used to test the cube bg model production and can be compared to
+each other using the plot_bg_cube_model_comparison.py example script.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals) # python 2 as python 3
+import os
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import Angle
+from astropy.time import Time
+from gammapy.datasets import make_test_bg_cube_model, make_test_dataset
+from gammapy.obs import (DataStore, ObservationGroups,
+                         ObservationGroupAxis)
+from gammapy.background import make_bg_cube_model
+
+
+#TEST = True # create a small dataset
+TEST = False # create a large dataset (slow)
+
+
+# using values from scripts.group_observations
+AZ_RANGE = Angle([90, 270], 'degree')
+ALT_RANGE = Angle([72, 90], 'degree')
+GROUP_ID = 27
+OUTDIR = 'bg_cube_models'
+OVERWRITE = False
+
+
+def create_dummy_observation_grouping():
+    """Define dummy observation grouping."""
+
+    altitude_edges = ALT_RANGE
+    azimuth_edges = AZ_RANGE
+    group_id = GROUP_ID
+
+    list_obs_group_axis = [ObservationGroupAxis('ALT',
+                                                altitude_edges,
+                                                'bin_edges'),
+                           ObservationGroupAxis('AZ',
+                                                azimuth_edges,
+                                                'bin_edges')]
+    obs_groups = ObservationGroups(list_obs_group_axis)
+    obs_groups.obs_groups_table['GROUP_ID'][0] = group_id
+
+    return obs_groups
+
+
+def make_true_model():
+    """Make a true bg cube model."""
+
+    overwrite = OVERWRITE
+
+    # create output folder
+    outdir = OUTDIR + '/true'
+    if not os.path.isdir(outdir):
+        os.mkdir(outdir)
+    else:
+        if overwrite:
+            # delete and create again
+            shutil.rmtree(outdir) # recursively
+            os.mkdir(outdir)
+        else:
+            # do not overwrite, hence exit
+            s_error = "Cannot continue: directory \'{}\' exists.".format(outdir)
+            raise RuntimeError(s_error)
+
+    # create dummy observation grouping
+    obs_groups = create_dummy_observation_grouping()
+    # save
+    outfile = outdir + '/bg_observation_groups.ecsv'
+    print('Writing {}'.format(outfile, overwrite=overwrite))
+    obs_groups.write(outfile)
+
+    # use binning from CubeBackgroundModel.define_cube_binning
+    detx_range = (Angle(-0.07, 'radian').to('degree'),
```

Comment:
If you're using the same binning for x and y, just have one `det_range` variable and use it twice in the call below.